data-prepper-plugins/key-value-processor/src/main/java/org/opensearch/dataprepper/plugins/processor/keyvalue/KeyValueProcessorConfig.java

```diff
@@ -99,45 +121,71 @@ public class KeyValueProcessorConfig {
             "The value specified in this configuration is the value used in <code>destination</code> map. " +
             "The default behavior is to drop the key-value pair.")
+    @ExampleValues({
+        @Example(value = "none", description = "key1value1&key2=value2 parses into {\"key1value1\": \"none\", \"key2\": \"value2\"}.")
+    })
     private Object nonMatchValue = DEFAULT_NON_MATCH_VALUE;
 
     @JsonProperty(value = "include_keys", defaultValue = "[]")
     @JsonPropertyDescription("An array specifying the keys that should be included in the destination field. " +
             "By default, all keys will be added.")
     @NotNull
+    @ExampleValues({
+        @Example(value = "[\"key2\"]", description = "key1=value1&key2=value2 will parse into {\"key2\": \"value2\"}.")
+    })
     private List<String> includeKeys = DEFAULT_INCLUDE_KEYS;
 
     @JsonProperty(value = "exclude_keys", defaultValue = "[]")
     @JsonPropertyDescription("An array specifying the parsed keys that should be excluded from the destination field. " +
             "By default, no keys will be excluded.")
     @NotNull
+    @ExampleValues({
+        @Example(value = "[\"key2\"]", description = "key1=value1&key2=value2 will parse into {\"key1\": \"value1\"}.")
+    })
     private List<String> excludeKeys = DEFAULT_EXCLUDE_KEYS;
 
     @JsonPropertyDescription("A prefix to append before all keys. By default no prefix is added.")
+    @ExampleValues({
+        @Example(value = "custom", description = "{\"key1=value1\"} parses into {\"customkey1\": \"value1\"}.")
+    })
     private String prefix = null;
 
     @JsonProperty("delete_key_regex")
     @JsonPropertyDescription("A regular expression specifying characters to delete from the key. " +
             "Special regular expression characters such as <code>[</code> and <code>]</code> must be escaped with <code>\\\\</code>. " +
             "Cannot be an empty string. " +
             "By default, no characters are deleted from the key.")
+    @ExampleValues({
+        @Example(value = "\\s", description = "{\"key1 =value1\"} parses into {\"key1\": \"value1\"}.")
+    })
     private String deleteKeyRegex;
 
     @JsonProperty("delete_value_regex")
     @JsonPropertyDescription("A regular expression specifying characters to delete from the value. " +
             "Special regular expression characters such as <code>[</code> and <code>]</code> must be escaped with <code>\\\\</code>. " +
             "Cannot be an empty string. " +
             "By default, no characters are deleted from the value.")
+    @ExampleValues({
+        @Example(value = "\\s", description = "{\"key1=value1 \"} parses into {\"key1\": \"value1\"}.")
+    })
     private String deleteValueRegex;
 
     @JsonProperty(value = "transform_key", defaultValue = "none")
     @JsonPropertyDescription("Allows transforming the key's name such as making the name all lowercase.")
+    @ExampleValues({
```

Comment:
Let's avoid sample values on enumerations. Those are already available in the enum list.